modules/network-commatrix-plugin.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/configuring-firewall.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-commatrix-plugin_{context}"]
+= Generating ingress network flow data using the `commatrix` plugin
+
+The `commatrix` plugin for the `oc` command generates ingress network flow data from your cluster. You can also use the plugin to identify any differences between open ports on the host and expected ingress flows for your environment.
+
+The plugin generates ingress flows to {product-title} services for the following environments:
+
+* {product-title} on bare metal
+* {sno-caps} with other platforms
+* {product-title} on {aws-first}
+* {sno-caps} on {aws-short}
+
+The plugin outputs the network flow data in various formats, such as CSV or JSON.
+
+.Prerequisites
+* You installed the OpenShift CLI (`oc`).
+* You logged in as a user with `cluster-admin` privileges.
+* You installed the `Krew` plugin manager for `oc`. For more information, see "Installing Krew".
+* You installed Go for building the plugin, or you downloaded a pre-built Go binary.
+* You installed `git` for cloning the plugin repository.
+
+.Procedure
+
+. Install the `commatrix plugin` by using `Krew`. You can point `Krew` to the manifest file for the plugin by cloning the `commatrix` Github repository locally or by pointing `Krew` to the remote repository manifest file:
+
+.. To install the plugin by cloning the repository, run the following commands:
++
+[source,bash]
+----
+$ git clone https://github.com/openshift-kni/commatrix.git
```

Comment:
you can write anything on paper
but are we seriously going to officially support and instruct people to clone upstream github and compile on their own?

this is error prone in so many ways..